### ISSUE TITLE
update version strings

### ISSUE THREAD

**Issue by E3V3A** [kou1okada/apt-cyg#28]
Hi!
Can you please update the version strings to reflect your fork?
There is no point in putting the copyright here, and would be more useful to say this is your version.

I'm using 3 different versions of apt-cyg to ensure stuff works, yours is one of them I really like,
but I'm always confused which one is which, without having to rename them specifically. 

```bash
function version()
{
  echo "apt-cyg version 0.57"
  echo "Written by Stephen Jungels"
  echo ""
  echo "Copyright (c) 2005-9 Stephen Jungels.  Released under the GPL."
}
```

**Comment by mettacrawler** [comment]
I don't use Cygwin any more, sorry.

**Comment by kou1okada** [comment]
Don't worry, it's not your work, @mettacrawler.

**Comment by kou1okada** [comment]
I wrote 346a752cbea02057346d3a55fdc47a5a5800ab79 on [branch: issue-28](https://github.com/kou1okada/apt-cyg/tree/issue-28).

Example of result.
~~~
$ apt-cyg --version
kou1okada/apt-cyg
forked from transcode-open/apt-cyg
origin: https://github.com/kou1okada/apt-cyg.git
commit: 2018-11-07 19:02:30 +0900 346a752 (HEAD -> issue-28, origin/issue-28)

This script is based on apt-cyg version 0.57
Copyright (c) 2005-9 Stephen Jungels. Released under the GPL.
Copyright (c) 2013-7 Stephen Jungels. Republished under the MIT license.
~~~

**Comment by E3V3A** [comment]
Very nice!

**Comment by E3V3A** [comment]
Don't forget to merge PR...

**Comment by kou1okada** [comment]
Now, I merged 346a752cbea02057346d3a55fdc47a5a5800ab79 to master.
